### PR TITLE
Merging the feature branch

### PR DESCRIPTION
**Pull request by jobyko**
Feature branch contains : below changes 

1.  JPG to PNG conversion.
2.  Link correction.
3.  Update projects in "our oss projects"

@asa1997  Please check and share your comments.

### CODE REVIEW

File: docs/bes-open-source-security-tool-tech-stack.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ hide:
 
 <h1> Open Source Security Tools [S] Tech Stack </h1>
```
Comment (asa1997) [inline docs/bes-open-source-security-tool-tech-stack.md:11]: Please raise a pull from base repo first. I have made some changes in this file.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@jobyko Please address the comment.

File: docs/bes-open-source-security-tool-tech-stack.md (hunk 1 above)
Comment (jobyko) [reply docs/bes-open-source-security-tool-tech-stack.md:11]: @asa1997 
Its already done, pull request is from synced feature branch (fb-3).

**Review by asa1997 — APPROVED** [review]
@jobyko @panickervinod LGTM.

**Comment by panickervinod** [comment]
@jobyko @asa1997 Merging this one. 

However the subject line " i.e. "Merging the feature branch""  is not unique. It can mention the main issue which is getting addressed by this pull request. Going forward we must start introducing Dewy notation for features along with RF numbers & ref issue numbers in the issue tracker where applicable..

**Pull request merged**